zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/ParagraphTest.java

```diff
@@ -232,4 +226,11 @@ public void returnUnchangedResultsWithDifferentUser() throws Throwable {
 
 
   }
+
+  @Test
+  public void deserializeParagraphFromDifferentDateFormat() {
+    Paragraph paragraph = Note.GSON
+        .fromJson("{\"dateUpdated\":\"2017-09-01 00:01:02.345\"}", Paragraph.class);
+    System.out.println(paragraph.dateUpdated);
```

Comment:
Can we have a better unit test for that? i dont think `System.out.println` is enough.
can we use something like `assert`?